src/mavlink-router/mainloop.cpp

```diff
@@ -63,6 +66,8 @@ Mainloop &Mainloop::init()
     return _instance;
 }
 
+static const char* pipe_path = "/tmp/mavlink_router_pipe";
```

Comment:
/tmp is usually not the proper place for a named pipe. /var/run/mavlink-router would probably be better.

Reply:
So, the reason it is in /tmp/ is so that for development purposes you don't need to run it as root or do any special setup. Thoughts? I guess this points more strongly towards using a TCP port...

Reply:
the setup is one time only: create the dir with the appropriate perms. We can also use configure to set it to another place for convenience for developers... Example:  ./configure  --rundir=/tmp

Reply:
Also, as I said in previous version of this patch, we should not use a pipe.. it should rather be implemented with a unix/tcp socket.